### PR TITLE
docs: Update to new Tiltfile syntax

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: docs/first_config.md [hunk 1]
```diff
@@ -26,14 +23,32 @@ A `Tiltfile` uses a small subset of Python called
...
+ADD . .
+RUN go install github.com/windmilleng/integration/oneup
+ENTRYPOINT /go/bin/oneup
+```
+
+If you don't know Go, that's OK. These are steps to run to build a Go server.
+
+- `k8s_resource('oneup', 'oneup.yaml')`
+
+This next line reads a Kubernetes YAML, gives it a name and creates it in Kubernetes. Tilt tracks while files you read, then automatically
```
Comment (dbentley) [inline docs/first_config.md]: s/"a Kubernetes YAML"/"Kubernetes YAML"/

I also think you can change "Tilt tracks while files you read" to "Tilt tracks dependencies; you can edit yaml or Dockerfile or Tiltfile and Tilt will ..."

File: docs/upgrade_skaffold.md [hunk 2]
```diff
@@ -64,25 +50,17 @@ deploy:
       - ./deployments/snack.yaml
 ```
 
-In Tilt we similarly associate your image and your Kubernetes YAML through the concept of a service:
+In Tilt we associate your image and your Kubernetes YAML by image tag.
 
 ```python
-  yaml = read_file('./deployments/snack.yaml')
-  service = k8s_service(img, yaml=yaml)
+k8s_resource('snack', 'deployments/snack.yaml')
```
Comment (dbentley) [inline docs/upgrade_skaffold.md]: I think this has made enough room that it'd be more instructive to include:
*) multiple dockerfiles
*) multiple yamls
*) all passed to k8s_resource("deploy', ['snack.yaml', 'vigoda.yaml'])

WDYT?

File: docs/upgrade_skaffold.md [hunk 3]
```diff
@@ -64,25 +50,17 @@ deploy:
...
 ```
 
-5. Return your service!
-
-```python
-  return service
-```
+In Skaffold services are implicitly named from their Kubernetes config. In Tilt services have names given by the first argument to the `k8s_resource` function.
```
Comment (dbentley) [inline docs/upgrade_skaffold.md]: I'd say "Tilt breaks out resources by image."

I think skaffold does it be pod name, but now I realize I'm not sure...

**Comment by jazzdan** [comment]
Updated! @dbentley

**Pull request merged**